packages/client/src/object/Cache.test.ts

```diff
@@ -89,20 +89,14 @@ describe("AsyncCache", () => {
   });
 
   describe("race checks", () => {
-    let factoryFn: Mock<[MinimalClient, string], Promise<any>>;
+    let factoryFn: Mock<AsyncFactory<any, any>>;
```

Comment:
Honestly, this is a way better type api for vitest (taking the typeof a function instead of an array of args and a return value). Alas, it required minor changes in this file